main.go

```diff
@@ -56,8 +56,8 @@ func CreateHandler(conf Config, git Git, pullRequests PullRequests, repositories
 	}
 }
 
-// startsWithPlusOne matches strings that start with either a +1 (not followed by other digits) or a :+1: emoji
-var startsWithPlusOne = regexp.MustCompile(`^(:\+1:|\+1($|\D))`)
+// isPlusOneComment matches strings that contain either a +1 (not followed by other digits) or a :+1: emoji
+var isPlusOneComment = regexp.MustCompile(`\+1($|\D)`)
```

Comment:
It does, good point. I took the approach of duplicating the current tests for it, but testing it extensively in a unit test and keeping a single "integration" (quoted, because I'm not sure what the correct word is for the current tests) test is a better approach.